appservice/src/createAppService/createWebApp.ts

```diff
@@ -30,6 +31,9 @@ export async function setWizardContextDefaults(wizardContext: IAppServiceWizardC
         } else if (await fse.pathExists(path.join(fsPath, 'requirements.txt'))) {
             // requirements.txt are used to pip install so a good way to determine it's a Python app
             wizardContext.recommendedSiteRuntime = LinuxRuntimes.python;
+        } else if (await javaUtils.isJavaProject(fsPath)) {
+            wizardContext.newSiteOS = WebsiteOS.linux;
+            wizardContext.newPlanSku = { name: 'P1v2', tier: 'PremiumV2', size: 'P1v2', family: 'P', capacity: 1 };
```

Comment:
On a related note - do you want to leverage `recommendedSiteRuntime`? You'd have to refactor a bit since I assume you want to recommend multiple runtimes (java, tomcat, and wildfly), but that could be done. Just a question - not required for this PR.

Reply:
Yes, I was intended to address it in another PR.